core/src/main/c/share/ooo.cpp

```diff
@@ -238,6 +226,7 @@ inline void swap(index_t *a, index_t *b) {
  *  of pivot
  *
  **/
+__SIMD_MULTIVERSION__
```

Comment:
we are not using GCC multiversion? should we perhaps tidy this up?

Reply:
ok